scripts/actions/mobskills/smite_of_rage.lua

```diff
@@ -9,11 +9,10 @@ mobskillObject.onMobSkillCheck = function(target, mob, skill)
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
-    local numhits = 1
     local accmod = 1
-    local dmgmod = 2.7
-    local info = xi.mobskills.mobPhysicalMove(mob, target, skill, numhits, accmod, dmgmod, xi.mobskills.magicalTpBonus.NO_EFFECT)
-    local dmg = xi.mobskills.mobFinalAdjustments(info.dmg, mob, skill, target, xi.attackType.PHYSICAL, xi.damageType.SLASHING, info.hitslanded * math.random(2, 3))
+    local dmgmod = .03
+    local info = xi.mobskills.mobPhysicalMove(mob, target, skill, mob:getWeaponDmg(), accmod, dmgmod, xi.mobskills.magicalTpBonus.NO_EFFECT)
```

Comment:
I recall a previous discussion (i can't seem to find the PR where) we would be getting rid of numhits in favor of mob:getWeaponDmg() * multiplier...

If you look at mobskills zarbzan, wind_blade, or stormwind, for example, you'll see numhits is not present.

Reply:
i can certainly add it back, but its unused right?

Reply:
Those are magical moves, not physical ones.
They use `xi.mobskills.mobMagicalMove` instead of `xi.mobskills.mobPhysicalMove`

numhits is certanly used in the physical mobskill function
![imagen](https://github.com/LandSandBoat/server/assets/60053999/edb9e0e4-7ec2-4d5d-849f-243b26837ae1)

Reply:
That's the confusion here.
`xi.mobskills.mobMagicalMove` does indeed use `mob:getWeaponDmg() * multiplier` in the place of numhits, because magicalmoves are always single hit iirc.